src/linttest/testdata/mustache/golden.txt

```diff
@@ -52,6 +52,21 @@ MAYBE   phpdoc: Missing PHPDoc for "getHelperName" public method at testdata/mus
 MAYBE   phpdoc: Missing PHPDoc for "getTemplateName" public method at testdata/mustache/src/Mustache/Exception/UnknownTemplateException.php:34
     public function getTemplateName()
                     ^^^^^^^^^^^^^^^
+WARNING bareReturn: Replace 'return' with 'return null' at testdata/mustache/src/Mustache/Parser.php:207
```

Comment:
return; and return null; already mean the same thing. Maybe provide more context here on why exactly do you ask to replace it in this function?

Reply:
Probably we can provide additional information after the first sentence?
For example:
```
<critical> WARNING bareReturn: Replace 'return' with 'return null'. The '\f' function has @return annotation with non-'void' value, so 'return null' is preferred. at C:/projects/noverify/example/index.php:7
    return;
    ^^^^^^^
```